helix-term/src/ui/editor.rs

```diff
@@ -578,6 +578,40 @@ impl EditorView {
         }
     }
 
+    /// Render bufferline at the top
+    pub fn render_bufferline(editor: &Editor, viewport: Rect, surface: &mut Surface) {
+        let pb = PathBuf::from(SCRATCH_BUFFER_NAME); // default filename to use for scratch buffer
```

Comment:
this is a bit of a nit but this variable name could be more descriptive: `scratch` or maybe `scratch_path`